src/renderer/views/Watch/Watch.vue

```diff
@@ -54,7 +54,7 @@
               v-if="upcomingTimestamp !== null"
               class="premiereText"
             >
-              {{ $t("Video.Premieres on") }}:
+              {{ $t("Video.Premieres in") }} {{ upcomingTimeLeft }}
```

Comment:
So it's possible to have `Premieres in Starting soon`?

Reply:
I guess I forgot to test that, that would be possible and would look weird.

After testing it with a video that was about to go live I noticed that when time has passed the scheduled timestamp but the video is not yet live then `upcomingTimeStamp` will be null.
Now I have to wonder if it's possible to even see it in normal use cases.